src/components/book.js

```diff
@@ -1,13 +1,14 @@
 import PropTypes from 'prop-types';
 import './book.css';
 import { useDispatch } from 'react-redux';
-import { removeBook } from '../redux/books/books';
+import { deleteBook, removeBook } from '../redux/books/books';
 
 export default function Book(props) {
   const { id, title, category } = props;
   const dispatch = useDispatch();
   const remove = (book) => {
     dispatch(removeBook(book));
```

Comment:
OPTIONAL

- [ ] I suggest you use ES6 syntax in your app. Use arrow functions instead of the function keyword for all your functions.

Reply:
Hello,

I changed the functions into arrow functions as you suggested, but the applications run perfectly at my end, I tried to experience the error you noticed but I can not.

I'll wait to get more feedback about it, thanks.